### DISCUSSION TITLE
Indexer controller account balance warning

### DISCUSSION THREAD

**Discussion by jim-counter** [subquery/subql#983] [Frontier TestNet - Bug Reporting]
A few people (me included) have been faced with gas estimation errors in their logs when their controller account DEV balance runs too low. Would it be sensible to add a warning to the Account UI when this balance falls below a certain threshold. Or even an alert could be pushed in some way?

I appreciate the real threshold is open to interpretation/individual preference so maybe it should be set by the indexer themselves. I've found that <0.08 DEV on the testnet would seem a sensible starting point.

**Comment by mzxyz** [comment]
Thanks for the suggestion. We will check the controller balance before sending the transactions, also the admin app will display alert when the balance is not enough. Please expect this includes in the next release, thanks

**Comment by mzxyz** [comment]
Resolved in `coordinator:v0.1.10`
